dbi/models/auth-users/behaviors.js

```diff
@@ -137,7 +137,7 @@ module.exports = (models, cache) => {
    */
   behaviors.hooks.beforeCreate = [
     // Hash password.
-    (authUser, options) => models.AuthUser.hashPassword(authUser, options),
+    (authUser, options) => authUser.password && models.AuthUser.hashPassword(authUser, options),
```

Comment:
this should not allow them to not include a password, but instead should be able to be overridden.

`(authUser, options) => (options.requirePassword === false) || models.AuthUser.hashPassword(authUser, options),`

I think is going to cause less issues.